CI failure: ietf.secr.sreq.tests.SessionRequestTestCase.test_edit

### Describe the issue

In https://github.com/ietf-tools/datatracker/actions/runs/6316222819/job/17150373788?pr=6382
```
======================================================================
FAIL: test_edit (ietf.secr.sreq.tests.SessionRequestTestCase.test_edit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/datatracker/datatracker/ietf/secr/sreq/tests.py", line 168, in test_edit
    self.assertContains(r, 'Second session with: {} {}'.format(group3.acronym, group4.acronym))
  File "/usr/local/lib/python3.9/site-packages/django/test/testcases.py", line 660, in assertContains
    self.assertTrue(
AssertionError: False is not true : Couldn't find 'Second session with: acronym5281 acronym5283' in response
```

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

--- a/ietf/secr/sreq/tests.py
+++ b/ietf/secr/sreq/tests.py
@@ -81,9 +81,9 @@ class SessionRequestTestCase(TestCase):
     def test_edit(self):
         meeting = MeetingFactory(type_id='ietf', date=date_today())
         mars = RoleFactory(name_id='chair', person__user__username='marschairman', group__acronym='mars').group
-        group2 = GroupFactory()
-        group3 = GroupFactory()
-        group4 = GroupFactory()
+        group2 = GroupFactory(acronym='group2')
+        group3 = GroupFactory(acronym='group3')
+        group4 = GroupFactory(acronym='group4')
         iabprog = GroupFactory(type_id='program')
 
         SessionFactory(meeting=meeting,group=mars,status_id='sched')
@@ -144,6 +144,7 @@ class SessionRequestTestCase(TestCase):
         r = self.client.post(url, post_data, HTTP_HOST='example.com')
         redirect_url = reverse('ietf.secr.sreq.views.view', kwargs={'acronym': 'mars'})
         self.assertRedirects(r, redirect_url)
+        print(1, post_data)
 
         # Check whether updates were stored in the database
         sessions = Session.objects.filter(meeting=meeting, group=mars).order_by("id")  # order to match edit() view
@@ -162,6 +163,7 @@ class SessionRequestTestCase(TestCase):
 
         # Check whether the updated data is visible on the view page
         r = self.client.get(redirect_url)
+        print(2, r.content)
         self.assertContains(r, 'Schedule the sessions on subsequent days')
         self.assertContains(r, 'Thursday early afternoon, Thursday late afternoon')
         self.assertContains(r, group2.acronym)

--- a/ietf/secr/sreq/tests.py
+++ b/ietf/secr/sreq/tests.py
@@ -144,7 +144,6 @@ class SessionRequestTestCase(TestCase):
         r = self.client.post(url, post_data, HTTP_HOST='example.com')
         redirect_url = reverse('ietf.secr.sreq.views.view', kwargs={'acronym': 'mars'})
         self.assertRedirects(r, redirect_url)
-        print(1, post_data)
 
         # Check whether updates were stored in the database
         sessions = Session.objects.filter(meeting=meeting, group=mars).order_by("id")  # order to match edit() view
@@ -163,7 +162,6 @@ class SessionRequestTestCase(TestCase):
 
         # Check whether the updated data is visible on the view page
         r = self.client.get(redirect_url)
-        print(2, r.content)
         self.assertContains(r, 'Schedule the sessions on subsequent days')
         self.assertContains(r, 'Thursday early afternoon, Thursday late afternoon')
         self.assertContains(r, group2.acronym)

--- a/ietf/secr/sreq/tests.py
+++ b/ietf/secr/sreq/tests.py
@@ -81,9 +81,9 @@ class SessionRequestTestCase(TestCase):
     def test_edit(self):
         meeting = MeetingFactory(type_id='ietf', date=date_today())
         mars = RoleFactory(name_id='chair', person__user__username='marschairman', group__acronym='mars').group
-        group2 = GroupFactory(acronym='group2')
-        group3 = GroupFactory(acronym='group3')
-        group4 = GroupFactory(acronym='group4')
+        group2 = GroupFactory()
+        group3 = GroupFactory()
+        group4 = GroupFactory()
         iabprog = GroupFactory(type_id='program')
 
         SessionFactory(meeting=meeting,group=mars,status_id='sched')
@@ -165,7 +165,8 @@ class SessionRequestTestCase(TestCase):
         self.assertContains(r, 'Schedule the sessions on subsequent days')
         self.assertContains(r, 'Thursday early afternoon, Thursday late afternoon')
         self.assertContains(r, group2.acronym)
-        self.assertContains(r, 'Second session with: {} {}'.format(group3.acronym, group4.acronym))
+        # The sessions can be in any order in the HTML, deal with that
+        self.assertRegex(r.content.decode(), r'Second session with: ({} {}|{} {})'.format(group3.acronym, group4.acronym, group4.acronym, group3.acronym))
 
         # check that a notification was sent
         self.assertEqual(len(outbox), 1)

